docs/src/Modal.doc.js

```diff
@@ -528,6 +528,72 @@ function Example(props) {
   />
 );
 
+card(
+  <Example
+    name="Outside Event Exception Test"
+    description={`
+      Testing that outside event is working as expected with portals
+    `}
+    defaultCode={`
+function Example(props) {
+  const [showModal, setShowModal] = React.useState(false);
+  const [hasLoaded, setHasLoaded] = React.useState(false);
```

Comment:
@emilyalwayss can we remove `hasLoaded` - doesn't look like it's used in this example.